packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js

```diff
@@ -39,6 +39,7 @@ const documentTypeMap = {
 	[documentTypes.uploadOriginalApplicationForm.name]: 'originalApplicationForm',
 	[documentTypes.whoWasNotified.name]: 'whoNotified',
 	[documentTypes.conservationMap.name]: 'conservationMap',
+	[documentTypes.representationUpload.name]: 'otherPartyRepresentations',
 	[documentTypes.planningOfficersReportUpload.name]: 'planningOfficerReport'
```

Comment:
When we add the s78 appeal etc we're going to have to add quite a few doc types, maybe not one for now but at some point we might want to move this to its own file

Reply:
It should probably just be a property on documentTypes